src/client/app/shared/canvas/canvas.component.css

```diff
@@ -0,0 +1,5 @@
+svg#svg {
+  width: 800px;
+  height: 800px;
```

Comment:
There's your resizing problem right there. You're probably better off with a percent, or a flex, or even an absolute positioning here.